### PR TITLE
have writen tests to main.js

### PR DESCRIPTION
**Pull request by lexahnure**
I've written test for main.js with chai and sinon. Please check it, thanks.

### CODE REVIEW

File: hw11/src/main.test.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import chai from 'chai';
 import sinon from 'sinon';
 import { days, defaultProduct, money, users } from './constants';
-import { getDay, showMessage, getAdultUsers, getRandomUsers } from './main'
+import { getDay, showMessage, getAdultUsers, getRandomUsers, Product } from './main'
 
 const { expect } = chai;
 
 describe('getDay', () => {
+    const date = (new Date().getDay());
```
Comment (iliatcymbal) [inline hw11/src/main.test.js:9]: no need for round brackets

File: hw11/src/main.test.js [hunk 2]
```diff
@@ -28,33 +30,77 @@ describe('getAdultUsers', () => {
...
 
+    beforeEach(() => {
+        product = new Product;
     });
 
-    it('should return from 2 to 3', () => {
-        const testUsers = [{age: 20}, {age: 17}, {age: 21}, {age: 18}];
-        sinon.stub(Math, 'random').returns(0.1);
+    it('should throw error if no value', () => {
+        expect(() => { product.setPrice() }).to.throw(Error);
```
Comment (iliatcymbal) [inline hw11/src/main.test.js:88]: why need additional function? Just pass `product.setPrice`

File: hw11/src/main.test.js [hunk 3]
```diff
@@ -28,33 +30,77 @@ describe('getAdultUsers', () => {
...
+    });
 
-        expect(getRandomUsers(testUsers)).to.be.an('array').that.includes(testUsers[2], testUsers[3]);
+    it('should throw error if no value', () => {
```
Comment (iliatcymbal) [inline hw11/src/main.test.js:91]: description is not relevant to test

File: hw11/src/main.test.js [hunk 4]
```diff
@@ -28,33 +30,77 @@ describe('getAdultUsers', () => {
...
+        const val = 15;
 
+        product.setPrice(val);
+        expect(product.value).to.equal(val);
     });
+
+    it('should throw error if no value', () => {
+        const val = 9;
```
Comment (iliatcymbal) [inline hw11/src/main.test.js:99]: description is not relevant to test

**Review by iliatcymbal — APPROVED** [review]
please, see my notes
in general, good

File: hw11/src/main.test.js (hunk 2 above)
Comment (lexahnure) [reply hw11/src/main.test.js:88]: I've tried in this way, but it doesn't work.

**Pull request closed without merging**